misc/demo/assets/app.js

```diff
@@ -47,6 +47,19 @@ angular.module('ui.bootstrap.demo', ['ui.bootstrap', 'plunker', 'ngTouch', 'ngAn
 .controller('DownloadCtrl', DownloadCtrl);
 
 function MainCtrl($scope, $http, $document, $uibModal, orderByFilter) {
+  $scope.latestVersion = false;
+
+  // Grab old version docs
+  $http.get('versions-mapping.json')
+    .then(function(result) {
+      $scope.latestVersion = true;
```

Comment:
Yes, it does that. I felt like it was a better idea to just have the menu on the latest version.

We don't want each version to have a .json because each versioned doc will have a dropdown with their own list of "old docs".

Let's invent numbers in here. We are in version 20 so we show docs from version 10 to version 19.  If we keep the .json with each update, they all will have their own ranges. So if we go to version 15, we will have a drop down from version 5 to version 14 and that is not what we want.

IIRC, we decided to version only the X latest versions. so as soon as we release a new version, we would remove the oldest one from versioning.

Anyway, .json is in the root because it will contain only one list of versioned docs and we don't want each copy to have their own.

The dropdown available (with the same .json file) is something to discuss. I would need to hack all versioned docs to have it.